completion/glossaries-extra.cwl

```diff
@@ -2556,8 +2556,8 @@ topicmcols
 \glsifcategoryattributehasitem{category}{attribute}{item}{true}{false}#*
 
 ### 11 bib2gls: Managing Reference Databases ###
-\glsxtrresourcefile{filename%file}

```

Comment:
`%file`actually means that the txs completer tries to complete filenames on disk.

Reply:
But the completer provides a list of full filenames. Neither is perfect.

Feel free to close this PR if you prefer filename completer.